### ISSUE TITLE
Report used JDK version

### ISSUE THREAD

**Issue by ennru** [lightbend/kalix-jvm-sdk#1076]
Most users will just keep the JDK version we provide in samples.
Over time this might change and we should keep track of used JDKs and their versions.

Refs https://github.com/lightbend/kalix/issues/4722, #1045

**Comment by patriknw** [comment]
I think we already populate that in the `serviceRuntime`, which is sent to the proxy in the discovery call.
https://github.com/lightbend/kalix-jvm-sdk/blob/3dc366495725ed066423e0347a2310e89d5b359e/sdk/java-sdk/src/main/scala/kalix/javasdk/impl/DiscoveryImpl.scala#L57-L58

**Comment by beritou** [comment]
> I think we already populate that in the `serviceRuntime`, which is sent to the proxy in the discovery call.

This is correct. This information is currently already being logged. Here's an example:

> Received Spec from user function: service name: [], service version: [], **service runtime: [OpenJDK Runtime Environment 11.0.11+9]**, protocol version: [1.0], sdk: [kalix-java-sdk-1.0.2], proxy version [1.0.12], project [007ef2b5-d428-4700-a598-33e02b600421], service [realtimetransitdemo], service-uid [dc1fb64e-c2d5-4566-ad87-5e6d0ae38187], env [prod], region [us-east1],

**Comment by ennru** [comment]
Ok, that's great. We "just" need to make Grafana parse it and report it -- and show it in our Kalix Metrics dashboard.
